packages/sources/bx-digital/src/transport/price.ts

```diff
@@ -0,0 +1,69 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { BaseEndpointTypes } from '../endpoint/price'
+
+export interface ResponseSchema {
+  securityId: string
+  lastModifiedTime: number
+  closingPrice: string
+}
+
+export type HttpTransportTypes = BaseEndpointTypes & {
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema[]
+  }
+}
+export const httpTransport = new HttpTransport<HttpTransportTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      return {
+        params: [param],
+        request: {
+          baseURL: config.API_ENDPOINT,
+          headers: {
+            'API-key': config.API_KEY,
+          },
+        },
+      }
+    })
+  },
+  parseResponse: (params, response) => {
+    if (!response.data) {
+      return params.map((param) => {
+        return {
+          params: param,
+          response: {
+            errorMessage: `The data provider didn't return any value`,
+            statusCode: 502,
+          },
+        }
+      })
+    }
+
+    return params.map((param) => {
+      const security = response.data.find((r) => r.securityId == param.securityId)
+      if (security) {
+        return {
+          params: param,
+          response: {
+            result: Number(security.closingPrice),
```

Comment:
This could return `NaN` if `security.closingPrice` is undefined, might be worth including this in the `security` check for easier debugging